src/Entities/Collection.ts

```diff
@@ -34,6 +34,19 @@ export class Collection {
   @Column({ nullable: true })
   description?: string
 
+  @Field({
+    nullable: true,
+    description: "Markdown alternate of description field contents.",
+  })
+  @Column({ nullable: true })
+  descriptionMarkdown?: string
```

Comment:
I've gotten this sort of feedback a couple of times when updating MP so I'm going to put this out there for thought —

Since `description` and `descriptionMarkdown` are basically two representations of the same content, I wonder if it wouldn't be more idiomatic to modify the existing field so that…

```graphql
collection(slug: "foo") {
  description # retrieves the html, to preserve compat with existing clients
}

collection(slug: "foo") {
  description(format: "markdown") # retrieves the markdown-flavored version
}
```

(cc @mzikherman who's caught this kind of thing before for me)

If that's tricky for some reason given the `type-graphql` / `typeorm` setup here, then I don't think it's the end of the world.

Reply:
I have the sense we are phasing the HTML out, right? Perhaps we will eventually deprecate `description` or make it point directly to MD stuff?

Not averse to this, but not sure it's necessary?

Reply:
Yeah, I reckon that's fine. We will presumably have the cleansing fire of `ArtworkProductGrouping` at some point in the near future in any case.